serving/backend/calculate_level.py

```diff
@@ -1,9 +1,15 @@
 import math
+import os
+import sys
 
 import pandas as pd
 
 
-def _round_monster_level(level: float, threshold=0.33) -> int:
+sys.path.insert(0, os.sep.join(os.path.normpath(__file__).split(os.sep)[:-1]))
```

Comment:
Co tu się stało?

Reply:
To jakieś pozostałości po tym jak mi coś nie chciało działać 😓 😅

Reply:
Już pamiętam
Potrzebowałam coś zrobić tak, żeby działało w notebooku i w serverze

Reply:
Tak podejrzewałem, aczkolwiek to się rozwiązuje na inne sposoby, nie taką magią. Przede wszystkim powinno działać w serwerze, a w notebooku można robić inne cuda. W szczególności podejrzewam, że był problem z importami, tak? W PyCharmie można oznaczyć katalog jako Sources Root (prawy klawisz na katalog w projekcie -> mark as -> sources root), będzie na niebiesko, i będzie wtedy dodany do PYTHONPATH, to zwykle rozwiązuje takie problemy